bears/general/SpaceConsistencyBear.py

```diff
@@ -89,6 +110,7 @@ def run(self,
                     diffs={filename: diff},
                     file=filename,
                     line=line_number,
-                    additional_info='\n\n'.join(additional_info_texts))
+                    additional_info='\n\n'.join(additional_info_texts),
+                    aspect=Formatting('py'))
```

Comment:
Put `aspect` above the `additional_info` so you don't need to modify unnecessary lines. It make `git blame/bisect` and history much cleaner.